Update list item style of `HTMLTextAreaElement` (#31342)
mdn/content: files/en-us/web/api/htmltextareaelement/index.md

- {{domxref("HTMLTextAreaElement.defaultValue", "defaultValue")}}
  - : A string that represents the control's default value, which behaves like the {{domxref("Node.textContent")}} property.
- {{domxref("HTMLTextAreaElement.dirName", "dirName")}}
  - : A string that represents the directionality of the element.
- {{domxref("HTMLTextAreaElement.disabled", "disabled")}}
  - : A boolean that represents the element's [`disabled`](/en-US/docs/Web/HTML/Element/textarea#disabled) attribute, indicating that the control is not available for interaction.
- {{domxref("HTMLTextAreaElement.form", "form")}} {{ReadOnlyInline}}
  - : Returns a reference to the parent form element. If this element is not contained in a form element, it can be the [`id`](/en-US/docs/Web/HTML/Element/form#id) attribute of any {{HTMLElement("form")}} element in the same document or the value `null`.
- {{domxref("HTMLTextAreaElement.labels", "labels")}} {{ReadOnlyInline}}
  - : Returns a {{domxref("NodeList")}} of the {{HTMLElement("label")}} elements associated with this element.
- {{domxref("HTMLTextAreaElement.maxLength", "maxLength")}}

>   - : A long that represents the element's [`maxlength`](/en-US/docs/Web/HTML/Element/textarea#maxlength) attribute, indicating the maximum number of characters the user can enter. This constraint is evaluated only when the value changes.

Reviewer: ```suggestion
  - : A number that represents the element's [`maxlength`](/en-US/docs/Web/HTML/Element/textarea#maxlength) attribute, indicating the maximum number of characters the user can enter. This constraint is evaluated only when the value changes.
```

- {{domxref("HTMLTextAreaElement.readOnly", "readOnly")}}
  - : A boolean that represents the element's [`readonly`](/en-US/docs/Web/HTML/Element/textarea#readonly) attribute, indicating that the user cannot modify the value of the control.
- {{domxref("HTMLTextAreaElement.required", "required")}}
  - : A boolean that represents the element's [`required`](/en-US/docs/Web/HTML/Element/textarea#required) attribute, indicating that the user must specify a value before submitting the form.
- {{domxref("HTMLTextAreaElement.rows", "rows")}}
  - : An unsigned long that represents the element's [`rows`](/en-US/docs/Web/HTML/Element/textarea#rows) attribute, indicating the number of visible text lines for the control.
- {{domxref("HTMLTextAreaElement.selectionDirection", "selectionDirection")}}
  - : A string that represents the direction in which selection occurred. This is `forward` if selection was performed in the start-to-end direction of the current locale, or `backward` for the opposite direction. This can also be `none` if the direction is unknown.
- {{domxref("HTMLTextAreaElement.selectionEnd", "selectionEnd")}}

>   - : An unsigned long that represents the index of the end of selected text. If no text is selected, contains the index of the character that follows the input cursor. On being set, the control behaves as if `setSelectionRange()` had been called with this as the second argument, and `selectionStart` as the first argument.

Reviewer: ```suggestion
  - : A number that represents the index of the end of selected text. If no text is selected, it contains the index of the character that follows the input cursor. On being set, the control behaves as if `setSelectionRange()` had been called with this as the second argument, and `selectionStart` as the first argument.
```

- {{domxref("HTMLTextAreaElement.selectionStart", "selectionStart")}}

>   - : An unsigned long that represents the index of the beginning of selected text. If no text is selected, contains the index of the character that follows the input cursor. On being set, the control behaves as if `setSelectionRange()` had been called with this as the first argument, and `selectionEnd` as the second argument.

Reviewer: ```suggestion
  - : A number that represents the index of the beginning of selected text. If no text is selected, it contains the index of the character that follows the input cursor. On being set, the control behaves as if `setSelectionRange()` had been called with this as the first argument and `selectionEnd` as the second argument.
```

- {{domxref("HTMLTextAreaElement.validity", "validity")}} {{ReadOnlyInline}}

>   - : Returns the validity states that this element is in.

Reviewer: ```suggestion
  - : Returns the validity state that this element is in.
```
